src/System.Private.ServiceModel/tools/test/SelfHostWcfService/TestResources/RequestLoggingServiceBehavior.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I separated the PR into two commits. One has product code changes, the other has test code changes. That way any cherry picking won't pick up any test related code.